aws/src/main/java/org/apache/iceberg/aws/s3/S3OutputStream.java

```diff
@@ -74,13 +77,17 @@
   private final AwsProperties awsProperties;
 
   private CountingOutputStream stream;
-  private final List<File> stagingFiles = Lists.newArrayList();
+  private final List<FileWithEtag> stagingFilesWithETags = Lists.newArrayList();
```

Comment:
I think we can avoid changing this variable name because the class name is already self-explanatory